### ISSUE TITLE
Pardot v4 integration not sending `Pardot-Business-Unit-Id` header with authorization request.

### ISSUE THREAD

**Issue by tylerfowle** [solspace/craft3-freeform#1481]
### What happened?

after trying to authorize Pardot CRM integration I receive the following error

### Errors and Stack Trace (if available)

```shell
{"@attributes":{"stat":"fail","version":1,"err_code":181},"err":"A required header is missing: Pardot-Business-Unit-Id header not found on request."}
```

### How can we reproduce this?

1. Freeform > Settings > CRM > New Integration
1.a) setup integration in salesforce  
* disable PKCE
* scopes: (api, web, chatter_api, refresh_token, offline_access, pardot_api)
2. Select "Pardot v4" from dropdown - enter business unit id, client id, client secret
3. click save, approve oauth, return to /integrations/crm with "not able to authorize" error

### Freeform Edition

Pro

### Freeform Version

5.5.7

### Craft Version

4.9.7

**Comment by kjmartens** [comment]
Sorry for the delay @tylerfowle. We'll check into this on our end shortly.

**Comment by kjmartens** [comment]
This should be resolved now in Freeform 5.5.8. 🙂
